CMakeLists.txt

```diff
@@ -171,6 +171,25 @@ INSTALL(DIRECTORY ${CMAKE_BINARY_DIR}/lib DESTINATION ${CMAKE_INSTALL_PREFIX} PA
 
 INSTALL(CODE "execute_process(COMMAND python setup.py install --prefix=${CMAKE_INSTALL_PREFIX} WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/python)")
 
+IF (USE_PYTHON)
+	INSTALL(FILES ${CMAKE_BINARY_DIR}/python/setup.py 
+		DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/singa/python
+		COMPONENT PySinga)
+	INSTALL(FILES ${CMAKE_BINARY_DIR}/python/singa/singa_wrap.py
+		DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/singa/python/singa
+		COMPONENT PySinga)
+	INSTALL(FILES ${CMAKE_BINARY_DIR}/python/singa/_singa_wrap.so
+		DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/singa/python/singa
+		COMPONENT PySinga)
+	INSTALL(DIRECTORY ${PROJECT_SOURCE_DIR}/python 
```

Comment:
This command copies the files in the **build machine** not in the **target machine** as we discussed in #309. 

if you build the package with your modification, it will not work because it will be missing the Python files:

```
ar x python-singa-1.1.0.deb
tar -xvzf data.tar.gz 
```

I created this PR to add these lines again as they are deleted in the previous PR. Please do not remove them. If they cause a problem, please let me know what is the problem. The problem can not be that they are redundant because they are required.